### PR TITLE
Pushing data

### PR DESCRIPTION
**Pull request by abhi2434**
Pushing data to master to update the production site.

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -76,10 +77,57 @@ social:
...
+  - /assets/js/plugins/jquery.greedy-navigation.js
+  - /assets/js/plugins/jquery.magnific-popup.js
+  - /assets/js/plugins/jquery.smooth-scroll.min.js
+  - /assets/js/main.min.js
+home_banners             :
+  - name: "REST API Connectivity"
+    excerpt: "HTTP based endpoint connector is a pre-packaged software package which provides seamless connectivity to RESTful APIs and leverage to communicate with any application supporting standard REST protocol and standard authentication mechanisms. This adapter is App-specific and is limited to certain apps which are developed using HTTP Protocol and REST Webservices."
+    url : /connectors/Overview-of-GenericRest-Adapters/
+  - name: "Mapping and Transformation"
+    excerpt: "This document will help you to understand and work with the MAPPING and TRANSFORMATION functionality in APPSeCONNECT portal. Primarily transformation is the field where the corresponding schemas are to be mapped. This is also the place where the logic behind the transformation is implemented such that the integration proceeds uninterrupted."
```
Comment (soumitasengupta) [inline _config.yml]: These lined need to be changed.

File: _posts/API-Management/Quickstart/2018-09-14-Steps-to-register-url.md [hunk 2]
```diff
@@ -38,12 +38,11 @@ menus:
 
 ## Scenario of Registering URL 
 
-1: If the API Proxy or Webhook is to be executed on cloud, Registering the URL is must in such case.
+1: If the API Proxy or Webhook is to be executed on cloud, Registering the URL is must.
 
-2: If the user wants to keep its IP Public while running the webhook  from the OP agent. The URL is needed to be registered in order to enhance security in the transaction process. 
+2: Register URL is for enhancing security purpose
```
Comment (soumitasengupta) [inline _posts/API-Management/Quickstart/2018-09-14-Steps-to-register-url.md]: reviewed

File: _config.yml [hunk 3]
```diff
@@ -76,10 +77,57 @@ social:
...
 analytics:
-  provider               : "google" # false (default), "google", "google-universal", "custom"
+  provider               : "google-universal" # false (default), "google", "google-universal", "custom"
   google:
-    tracking_id          : UA-34675782-1
+    tracking_id          : UA-106094300-15
+  crazyegg:
+    url                  : 0089/9312.js
 
+footer_scripts           :
```
Comment (hiteshdey) [inline _config.yml]: Need Review

**Comment by abhi2434** [comment]
closing Pull request

**Comment by abhi2434** [comment]
Please merge the document locally and then push or commit.

**Pull request merged**